src/platform/packages/shared/as-code/filters-schema/kibana.jsonc

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "shared-common",
```

Comment:
Ideally, this would be `shared-server`, but Lens will need to use this in their schemas which are currently defined in a `shared-common` package.